source/blog/2019-05-11-rspec-rails-4-0-has-been-released.md

```diff
@@ -0,0 +1,181 @@
+---
+title: RSpec Rails 4.0 has been released!
+author: Benoit Tigeot, Jon Rowe and Phil Pirozhkov
+---
+
+RSpec Rails 4.0 has been released recently! Given our commitment to
+[semantic versioning](http://semver.org/), this update comes with 
```

Comment:
WDYT of:
"Given our commitment to [semantic versioning](http://semver.org/), we've been delaying the breaking changes until this update. One notable change is that we've dropped support for Rails below 4.2."